src/JunctionTreeTypes.jl

```diff
@@ -14,14 +14,16 @@ mutable struct BayesTree
   btid::Int
   cliques::Dict{Int,Graphs.ExVertex}
   frontals::Dict{Symbol,Int}
+  variableOrder::Vector{Symbol}
```

Comment:
I see how this is adding the variableOrder member variable, but it still fails the check with an error saying that BayesTree does not have that field.

Reply:
i committed a fix